cohere/client.py

```diff
@@ -192,6 +192,9 @@ def chat(
         temperature: float = 0.8,
         max_tokens: int = None,
         stream: bool = False,
+        p: float = None,
+        k: float = None,
+        logit_bias: Dict[int, float] = {},
```

Comment:
[empty dicts are a dangerous default value](https://stackoverflow.com/questions/26320899/why-is-the-empty-dictionary-a-dangerous-default-value-in-python). 
Typically you just use "None" here and have the API just handle that the same as {}